### DISCUSSION TITLE
Is it possible to fetch the timestamps of CSGO events such as player_death ?

### DISCUSSION THREAD

**Discussion by chipimix** [saul/demofile#248] [Questions] [answered]
Hi there!
Awesome project 👍 
Is it possible to fetch the timestamps of CSGO events such as player_death ?
Thanks in advance!

**Comment by Technoblazed** [comment, marked as answer]
`demoFile.currentTime` will give the current amount of seconds through the demo that the event occurs

**Comment by saul** [comment]
Hi @chipimix, please let us know if that didn't answer your question
